extensions/amp-video-iframe/0.1/amp-video-iframe.js

```diff
@@ -120,13 +120,18 @@ class AmpVideoIframe extends AMP.BaseElement {
   buildCallback() {
     const {element} = this;
 
-    this.user().assert(!isAdLike(element),
-        '<amp-video-iframe> does not allow ad iframes. ',
-        'Please use amp-ad instead.');
-
-    this.user().assert(!looksLikeTrackingIframe(element),
-        '<amp-video-iframe> does not allow tracking iframes. ',
-        'Please use amp-analytics instead.');
+    // TODO(alanorozco): On integration tests, `getLayoutBox` will returned
+    // a cached default value, which makes these assertions fail. Move to
+    // `describes.integration` to see if that fixes it.
+    if (!element.hasAttribute('i-amphtml-integration-test')) {
```

Comment:
Would `!getMode().test` work instead?

Reply:
That would be the case, but this is unit-tested as well, so we need to discern between the integration bug and the unit tests.

Reply:
I see